.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: true
+os: trusty
+cache: false
+
+env:
+  global:
+    - SFDX_URL=https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz
+
+    # SFDX_KEY_PASS
+    - secure: "PnBWbZDWwDiAYXxWPkmcnOEToixuzrO/qkzrW+BqvLz2fDlvKeEGR6oVlLuij1Mtz2bSHBZxiChHBs0hRQ118kJ3uitjYT/bvkILgJ/cgyH8VxSvhYBp09K3XXTCH/30rdrdlrj5wHfKCcN53clrmzPUsOTo/aa5g3U5dG/Ht5yqQ5SRyBvkCJmG2HeIn/ZAkaSFQTMquLrCJeAuw0HylqjTuvPsxVqXDS16E9bFeG0+D71roKlwP529zgpqb0eIQPJ0Ak5ezjxLmNFGI8+kltR/1Cdi+Mp+CVjCZV9D5jIv0SEln0Q1wJ2TR2MwyJdJIL8qyF/LVf6/+V/7qf8+6Yj6z1iJRFiuTsqbvY1AwdlNv5uK5qmC34xoUytugb9Htu8C1FsJqKGXCXunaNYeABmRtC9QNkjN1UPTBtSRDDfCGF6tH//LXlMNY0d/EHxFsOiZjI2trmTSRMGcURXPECel+eWkEsxgenYbBUUSnNfIAglW63zkew9emyUz2zwAE7H/eev/3RMGfKNy7/G0xhYNF4MnoOL815OKPcjVm8ZFncBBqY64BRDDsSTztg2ekfi31Iaz5wq8blDp8/0dph6Fedvc4+Tv4pFUKUyT8Ts2MwXiKSu67nCguPC0GYyPulKj1scGIQ+jI71b3lNJJZoluQm0UKnhWL/JNzbOVTs="
```

Comment:
should we be using vault here? https://github.com/rollbar/rollbar-sf-apex/pull/15/files#diff-354f30a63fb0907d4ad57269548329e3R10  Or are we keeping tokens in git for now?

Reply:
https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml